xwiki-platform-core/xwiki-platform-web/src/test/java/org/xwiki/web/DocumentTagsTest.java

```diff
@@ -75,7 +75,7 @@ void displayTagsWhenNoEditRightsAndTagPluginAvailableAndNoTags() throws Exceptio
         // - The tag label is displayed
         // - No tag is listed after the tag label
         // - No "+" link is displayed since the user doesn't have edit rights
-        assertThat(result, matchesPattern("\\Q<div class=\"doc-tags\" id=\"xdocTags\"> core.tags.list.label </div>"));
+        assertThat(result, matchesPattern("\\Q<div class=\"doc-tags hidden\" id=\"xdocTags\"> core.tags.list.label </div>"));
```

Comment:
You should also add a new test to prove that it works when there are tags.

Reply:
@CatalinStratu for me this is the only remaining point: making sure that it still works when there are tags. Please try to add a test and if you cannot, let me know.

Reply:
> @CatalinStratu for me this is the only remaining point: making sure that it still works when there are tags. Please try to add a test and if you cannot, let me know.

Hi, I tried to do a unittest but I can't, can you help me please?

Reply:
Could you show me the code that you wrote so far?

Reply:
I try to use this piece of code to try to test, but it doesn't work.
Source : https://github.com/xwiki/xwiki-platform/blob/0e3ec41cda7ac82f14e8cc092ff2e5f059ed1fc1/xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-ui/src/test/it/org/xwiki/test/ui/tag/AddRemoveTagsTest.java

![image](https://user-images.githubusercontent.com/28156781/113154950-35ea2900-9241-11eb-8489-28b59c225925.png)